scripts/cli/package.json

```diff
@@ -6,64 +6,78 @@
-    "3_auth_join_did": "node ./tests/3_auth_join_did.js",
-    "4_permission_management": "node ./tests/4_permission_management.js",
-    "5_claim_management": "node ./tests/5_claim_management.js",
-    "6_create_assets": "node ./tests/6_create_assets.js",
-    "7_create_claim_compliance": "node ./tests/7_create_claim_compliance.js",
-    "8_asset_transfer": "node ./tests/8_asset_transfer.js",
-    "9_bridge_transfer": "node ./tests/9_bridge_transfer.js",
-    "10_governance_management": "node ./tests/10_governance_management.js",
-    "11_A_settlement": "node ./tests/11_A_settlement.js",
-    "11_B_settlement": "node ./tests/11_B_settlement.js",
+    "tscov": "type-coverage --at-least 95",
+    "tscov:d": "type-coverage --detail",
+    "refresh": "yarn clean:all && yarn install",
+    "publish-package": "yarn build && yarn publish",
+    "docs": "typedoc src/tests src/helpers src/util --out dist/docs ",
+    "0_create_identities": "node ./dist/tests/0_create_identities.js",
+    "1_poly_transfer": "node ./dist/tests/1_poly_transfer.js",
+    "2_key_management": "node ./dist/tests/2_key_management.js",
+    "3_auth_join_did": "node ./dist/tests/3_auth_join_did.js",
+    "4_permission_management": "node ./dist/tests/4_permission_management.js",
+    "5_claim_management": "node ./dist/tests/5_claim_management.js",
+    "6_create_assets": "node ./dist/tests/6_create_assets.js",
+    "7_create_claim_compliance": "node ./dist/tests/7_create_claim_compliance.js",
+    "8_asset_minting": "node ./dist/tests/8_asset_minting.js",
+    "9_bridge_transfer": "node ./dist/tests/9_bridge_transfer.js",
+    "10_governance_management": "node ./dist/tests/10_governance_management.js",
+    "11_A_settlement": "node ./dist/tests/11_A_settlement.js",
+    "11_B_settlement": "node ./dist/tests/11_B_settlement.js",
     "schema_test": "./util/schema_test.sh",
-    "offchain_worker_test": "node ./util/offchain_worker_test.js",
+    "offchain_worker_test": "node ./dist/util/offchain_worker_test.js",
     "test": "./test.sh"
   },
   "author": "Polymath Network",
   "license": "UNLICENSED",
   "dependencies": {
-    "@liftr/tscov": "^2.0.0",
-    "@polkadot/api": "^3.7.1",
-    "@polkadot/util": "^5.5.1",
-    "@polkadot/util-crypto": "^5.5.1",
-    "@types/clear": "^0.1.1",
-    "@types/figlet": "^1.2.1",
-    "@types/inquirer": "^7.3.1",
-    "chai": "^4.2.0",
-    "chalk": "^4.1.0",
-    "clear": "^0.1.0",
-    "cli-progress": "^3.9.0",
-    "command-line-args": "^5.1.1",
-    "commander": "^7.0.0",
-    "crypto-random-string": "^3.3.0",
-    "esm": "^3.2.25",
-    "figlet": "^1.5.0",
-    "inquirer": "^7.3.3",
-    "minimist": "^1.2.5",
-    "nodejs-fs-utils": "^1.2.5",
-    "path": "^0.12.7",
+    "@polkadot/api": "3.7.1",
+    "@polkadot/util": "5.5.1",
+    "@polkadot/util-crypto": "5.5.1",
+    "@types/chai": "4.2.15",
+    "@types/clear": "0.1.1",
+    "@types/figlet": "1.2.1",
+    "@types/inquirer": "7.3.1",
```

Comment:
Type packages generally go under dev dependencies

Reply:
thanks, I will make the changes.